spec.bs

```diff
@@ -232,7 +232,105 @@ to conduct an auction to select an advertisement to display to the user, they ca
 {{Window/navigator}}.{{Navigator/runAdAuction()}} function, providing an auction configuration that
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
-  
+
+<h3 id="script-runners">Script Runners</h3>
+
+<xmp class="idl">
+[Exposed=Window, Global=Window]
+interface InterestGroupBiddingScriptRunnerGlobalScope{
+};
+
+[Exposed=Window, Global=Window]
+interface InterestGroupScoringScriptRunnerGlobalScope{
+};
+
+[Exposed=Window, Global=Window]
+interface InterestGroupReportingScriptRunnerGlobalScope{
+};
+</xmp>
+
+Issue: where do these interfaces above get exposed to?

```

Comment:
also, any suggested sentence about  the note of "webidl APIs with exposed=* will not be exposed to script runners?